Extensions/IISWebAppDeploy/Src/Tasks/SqlDacpacDeploy/DeployToSqlServer.ps1

```diff
@@ -0,0 +1,131 @@
+﻿Import-Module $env:CURRENT_TASK_ROOTDIR\DeploymentSDK\InvokeRemoteDeployment.ps1
+
+function EscapeSpecialChars
+{
+    param(
+        [string]$str
+    )
+
+    return $str.Replace('`', '``').Replace('"', '`"').Replace('$', '`$')
+}
+
+function IsPublishProfileEmpty
+{
+    param(
+        [string]$publishProfile
+    )
+
+    $publishProfile = $publishProfile.Trim('\', ' ').Trim()
```

Comment:
No it is a character '\', in powershell we represent that way only, ISE or PowerShell editor is showing properly only